README.md

```diff
@@ -5,10 +5,14 @@ Testausserverin siistein botti
 
 ## Ominaisuuksia
 
-### Haskell ajuri
-Kun pingaat kamelin viestiin, jossa on Haskell-koodipalikka
+### Kieliajurit
+Kun pingaat kamelin viestiin, jossa on tuettukieli-koodipalikka
```

Comment:
Im pretty sure "tuettukieli" is not a word. Just put something like "jossa on tuetun kielen koodipalikka"

Reply:
fixed